src/sidebar/services/import-annotations.ts

```diff
@@ -128,6 +128,7 @@ export class ImportAnnotationsService {
     this._store.beginImport(anns.length);
 
     const existingAnns = this._store.allAnnotations();
+    const currentUri = this._store.mainFrame()?.uri;
```

Comment:
I'm making a slightly blind guess that the main frame is the right one to get the URI

Reply:
For all the content types in the LMS that we support, this will be correct with the possible exception of VitalSource. You'll need to check what happens there.